addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -184,6 +184,17 @@ def _compute_session_duration_hour(self):
             )
             history.session_duration_hour = (end - history.create_date).total_seconds() / 3600
 
+    @api.model
+    def action_open_discuss_channel_view(self, domain=()):
```

Comment:
I talked with Seb about it, we can just modify the existing method and keep the old name. This is better because it avoids crashes if the JS is updated but the server hasn't been restarted yet. On the other hand, the naming confusion isn’t a big deal: in the context of a live chat report, it's always better to open the chat in Discuss when only one record is involved.